google/cloud/ndb/context.py

```diff
@@ -28,30 +28,36 @@
 from google.cloud.ndb import key as key_module
 
 
-def _generate_context_ids():
-    """Generate a sequence of context ids.
+class _ContextIds:
```

Comment:
Given Python 2.7, maybe:
```suggestion
class _ContextIds(object):
```

Reply:
That was a deliberate omission. I figure since Python 2.7 is getting severely deprecated in most contexts, and since old style classes are still available in 2.7, there's no point in introducing a Python 2-ism unless explicitly forced to. In this case there's nothing forcing the use of a new style class. If you prefer to subclass `object` anyway, though, I can change it.

Reply:
That was likely just a reflex response on my part -- I don't see anything in the class definition which would be affected by the differences between [old-style and new-style classes](https://portingguide.readthedocs.io/en/latest/classes.html).